packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -95,6 +95,23 @@ pub struct ExperimentConfig {
     clap(global = true, short = 'w', long, default_value_t = num_cpus::get(), validator = at_least_one, env = "HASH_WORKERS")
     )]
     pub num_workers: usize,
+
+    /// Allows a JavaScript runner to grow to some initial size in megabytes before triggering
+    /// garbage collections.
```

Comment:
```suggestion
    /// Allows the heap of the V8 runtime in a JavaScript runner to grow to some initial size in 
    /// megabytes before triggering garbage collections.
```

I think in a previous comment chain it was suggested that we shouldn't mention V8. I don't think I agree with this. We want to give user's the tools to do some debugging themselves, and importantly we want to mention that this is the **JavaScript** part of the **JavaScript** runner which I've always found a bit confusing. But we don't want users to think this includes the Rust heap '_of_' the runner.

Reply:
Can we also be explicit that providing `None` uses V8 defaults (**not** asking us to say what the defaults are, just that they're used)

Reply:
Specify default in https://github.com/hashintel/hash/pull/446/commits/68c782b0abc626407df5d7a2e9a16230f328ac50

Reply:
> I think in a previous comment chain it was suggested that we shouldn't mention V8

Yes, we previously only had mentioned "v8", but not "JavaScript runner". For me, it's important, that this explicitly states, that this affects the JS Runner. Mentioning "v8" **in addition** seems reasonable.

One other thing I noticed: Can we say "in **one** JavaScript runner" instead? Also, maybe add a comment to "--num-workers", that this implies the number of JS runners?

Reply:
I'd prefer **each** JavaScript runner rather than **a** or **one**

Reply:
Even better!

Reply:
Is 1 worker == 1 JS runner?
Or do they have another relation?

Reply:
1 worker _has_ 1 JS runner. A worker is made up of a JS Runner, a Python Runner, and soon a Rust Runner

Reply:
Is "num-workers" modifying the total number of workers or the number of workers that can run in parallel?
From the doc and default value I would guess the later but then how does it relate to this argument?

Reply:
Reference V8 and added mention of "num-workers" in https://github.com/hashintel/hash/pull/446/commits/781b90b6c2fe56ca925affac8b7be0d77b2ee863